app/Repositories/ReportbackRepository.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Rogue\Repositories;
+
+use Rogue\Models\Reportback;
+use Rogue\Models\ReportbackLog;
+
+class ReportbackRepository
+{
+    /**
+     * Create a new reportback
+     *
+     * @param  array $data
+     * @return \Gladiator\Models\Message
+     */
+    public function create($data)
+    {
+        // Store reportback
+        $reportback = new Reportback;
```

Comment:
Not a facade! 😉 Reportback is a real class with a static method.

The reason that facades bother me is the class (`\Auth` for example) doesn't _really_ exist, and Laravel then maps the "missing" class to a real one (`\Illuminate\Auth\AuthManager` or `\Illuminate\Contracts\Auth\Guard`) at runtime via PHP's Reflection API and the service container. So it can make it confusing to figure out what's happening behind the scenes.